internal/component/prometheus/remote/queue/filequeue/filequeue_bench_test.go

```diff
@@ -0,0 +1,50 @@
+package filequeue
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/go-kit/log"
+	"github.com/grafana/alloy/internal/component/prometheus/remote/queue/types"
+	"github.com/stretchr/testify/require"
+	"github.com/vladopajic/go-actor/actor"
+)
+
+func BenchmarkFileQueue(t *testing.B) {
```

Comment:
Nit: Can you add as a comment the current results of the benchmark for some future reference? I think it can be useful in certain cases.

Reply:
Going to drop this benchmark, its not super useful. There are better end to end benchmarks that track it better.